bugzooka/integrations/mcp_client.py

```
@@ -0,0 +1,39 @@
+        mcp_tools = await mcp_client.get_tools()
+        logger.info("MCP configuration loaded and tools retrieved.")
+
+    except FileNotFoundError:
+        logger.warning(
+            "MCP configuration file not found. Running without external tools."
+        )
+        mcp_tools = []
+        # Create a dummy client to avoid crashing, though it won't be used.
+        mcp_client = MultiServerMCPClient({})
+    except json.JSONDecodeError as e:
+        logger.error(f"Invalid JSON in MCP configuration file: {e}")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)